Makefile

```diff
@@ -10,13 +10,26 @@ SHELL := /bin/bash
 DATA_PATH ?= $(shell pwd)/data
 # Set the environment variable MJKEY with the contents of the file specified by
 # MJKEY_PATH.
-MJKEY_PATH ?= ~/.mujoco/mjkey.txt
+MJKEY_PATH ?= ${HOME}/.mujoco/mjkey.txt
+BUILD_ARGS = --build-arg user="${USER}" --build-arg uid=${shell id -u}
```

Comment:
look like maybe you can remove this variable, since it doesn't need to get overridden?